[2022-09-05 12:44:28,994] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: preprocessing_data.clean_data scheduled__2022-09-05T12:20:50.320155+00:00 [queued]>
[2022-09-05 12:44:29,000] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: preprocessing_data.clean_data scheduled__2022-09-05T12:20:50.320155+00:00 [queued]>
[2022-09-05 12:44:29,000] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 12:44:29,000] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-09-05 12:44:29,000] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 12:44:29,015] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): clean_data> on 2022-09-05 12:20:50.320155+00:00
[2022-09-05 12:44:29,021] {standard_task_runner.py:52} INFO - Started process 39033 to run task
[2022-09-05 12:44:29,025] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'preprocessing_data', 'clean_data', 'scheduled__2022-09-05T12:20:50.320155+00:00', '--job-id', '142', '--raw', '--subdir', 'DAGS_FOLDER/model_dag.py', '--cfg-path', '/tmp/tmpeu9l03ta', '--error-file', '/tmp/tmp5md4yv5p']
[2022-09-05 12:44:29,025] {standard_task_runner.py:80} INFO - Job 142: Subtask clean_data
[2022-09-05 12:44:29,076] {task_command.py:371} INFO - Running <TaskInstance: preprocessing_data.clean_data scheduled__2022-09-05T12:20:50.320155+00:00 [running]> on host rc1b-dataproc-m-xzxwmqcudfo0foh0.mdb.yandexcloud.net
[2022-09-05 12:44:29,130] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=preprocessing_data
AIRFLOW_CTX_TASK_ID=clean_data
AIRFLOW_CTX_EXECUTION_DATE=2022-09-05T12:20:50.320155+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-05T12:20:50.320155+00:00
[2022-09-05 12:45:06,166] {python.py:173} INFO - Done. Returned value was: None
[2022-09-05 12:45:06,174] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=preprocessing_data, task_id=clean_data, execution_date=20220905T122050, start_date=20220905T124428, end_date=20220905T124506
[2022-09-05 12:45:06,203] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-05 12:45:06,226] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
